_includes/components/header--basic.html

```diff
@@ -40,9 +40,9 @@
               {% endfor %}
             </ul>
             {% else %}
-            <a class="usa-nav-link{% if _section.href == page.permalink %} usa-current{% endif %}" href="{% if _section.external %}{{ _section.href }}{% else %}{{ _section.href | relative_url }}{% endif %}">
-              <span>{{ _section.text }}</span>
-            </a>
+              <a class="{% if _section.class %}{{ _section.class }}{% else %} usa-nav-link {% endif %} {% if _section.href == page.permalink %} usa-current{% endif %}" href="{% if _section.external %}{{ _section.href }}{% else %}{{ _section.href | relative_url }}{% endif %}">
```

Comment:
What do you think about just making this optional class in addition to the default class? `usa-nav-link` has some default padding/margin on it for proper alignment, so I could see that being useful to keep, and then maybe using the additional class to modify visual aspects, etc.

Reply:
We can try that out and see how they play together. I was thinking about the flexibility to be able to write your own class in without having to still have `usa-nav-link` in there 🤔

Reply:
Just thought of something else with this too, adding a class here without using a media query would affect the mobile navigation menu too. Are we making the assumption that if someone uses this class they'll be able to target the breakpoints properly? Maybe just a note about it in the documentation is enough?

Reply:
That's a good point, right now I'm targeting the buttons in the CSS here: https://github.com/18F/uswds-jekyll/blob/f59c5ca72a41beca36f3a48c956eae783c2f268a/_sass/nav-buttons.scss#L3

If you take a look at the preview link at small screens you can see they play nicely. 
https://federalist-proxy.app.cloud.gov/preview/18f/uswds-jekyll/sw-nav-button/